documentation/asciidoc/topics/proc_configuring_sites_automatically.adoc

```diff
@@ -12,14 +12,11 @@ Configure {ispn_operator} to establish cross-site views with {brandname} cluster
 
 . Create an `Infinispan` CR for each {brandname} cluster.
 . Specify the name of the local site with `spec.service.sites.local.name`.
-. Set the value of the `service.sites.local.expose.type` field to either `NodePort` or `LoadBalancer`.
+. Set the value of the `spec.service.sites.local.expose.type` field to either `NodePort` or `LoadBalancer`.
+. Configure ports with the following fields as appropriate:
```

Comment:
is it worth mention this is optional? I guess 99% of the users are fine with the default, especially in automatic xsite configuration.